### ISSUE TITLE
I have a question.

### ISSUE THREAD

**Issue by UdonDa** [LoreGoetschalckx/GANalyze#5]
Hi， @LoreGoetschalckx.
I have tried your implementation and it works well.

I have trained my model using these parameters.
```
python train.py \
    --generator biggan256 None  \
    --assessor emonet \
    --transformer OneDirection None \
    --train_alpha_a -0.5 \
    --train_alpha_b 0.5 \
    --gpu_id 0 \
    --num_samples 400000 \
    --checkpoint_resume 0
```

But, I have trouble with 3 points.

1. Given the negative alpha,  the model generates only grayscale images.
(here, https://github.com/LoreGoetschalckx/GANalyze/blob/master/pytorch/test_pytorch.py#L267 )

2. Given the positive alpha, the model generates only one similar sample.
I check the transformed noize in (https://github.com/LoreGoetschalckx/GANalyze/blob/master/pytorch/test_pytorch.py#L152) and notice each transformed z is a similar parameter.
I think I can not learn the model because the hyper parameter is wrong...

![n01692333 Gila monster, Heloderma suspectum](https://user-images.githubusercontent.com/25411643/75601431-602d5c80-5afe-11ea-8547-4667907c13d2.jpg)

3. Finally, I want to try other than the valence model.
Now, you open the only valence model of EmoNet (https://github.com/LoreGoetschalckx/GANalyze/blob/master/pytorch/download_pretrained.sh#L4), right?
If you do not mind, please open the other memorability, aesthetics, and emotional model.

Thanks.

**Comment by LoreGoetschalckx** [comment]
Hi @UdonDa ,

Thank you for your interest in our work. Could you let me know which value you used for the --alpha parameter of the test script? Am I right to assume you used the PyTorch version? Thanks!

**Comment by UdonDa** [comment]
Thank you for reply.
When I run the `test_pytorch.py`, I set the alpha 0.1.
```
python test.py \
    --alpha 0.1 \
    --test_truncation 1 \
    --checkpoint_dir ./checkpoints/biggan256_None/emonet/OneDirection_None/1 \
    --checkpoint 400000 \
    --gpu_id 1
```

**Comment by UdonDa** [comment]
When I run `test.py`, I check the parameter theta in https://github.com/LoreGoetschalckx/GANalyze/blob/master/pytorch/transformations/pytorch.py#L14

Then, I get the log
```
self.w: Parameter containing:
tensor([[ 19.8866,  -7.0352, -20.9344, -13.8647,   9.7017, -11.3761,  12.3890,
         -16.4173, -20.3028,  18.9324,  19.3871, -18.2960,  20.2939,  20.1526,
          20.4207,   9.8526,  -6.0939, -10.9522,  18.4912, -19.6304, -20.1626,
         -13.8126,  17.7707, -19.3851,  -0.1464,  15.8693,  19.5792,  20.2768,
          -8.4113,   5.7917,   9.4342,  19.8935,   9.5108,  -7.7949, -19.6066,
         -12.9996,  18.2787,  -6.9942, -20.4986,  21.5064, -21.7531,  14.1958,
          21.7881, -18.7075, -18.7224, -13.2258, -20.8932, -20.9823, -16.8899,
         -19.4925, -13.6387,  19.5383, -17.4751, -13.9109,   7.5406,  12.1912,
         -19.6123, -20.2762, -16.5088,   2.3638, -12.6745,   5.8565, -18.8860,
          19.4283,  10.1872, -21.4203,   1.3750,  21.2133,  14.8132, -20.3088,
         -21.8229,  21.1886,  -2.4047, -15.9073, -18.3353,  18.3123,  20.1681,
         -19.0842,  -1.3507, -18.0880,  -6.0686,  21.0491,  20.4660,  20.4645,
         -15.9493,  20.0604, -20.2166,  21.4947,   3.5261, -20.2525, -21.8708,
          18.5322,  12.2575,  19.0092,  19.3777, -16.1586, -21.4152,  20.5844,
         -20.1328, -18.4241,  -0.7067,  -6.6053,  -9.7084,  -1.6042,   1.4746,
          -9.5609,   0.3062,   3.8273,  -1.0951,   0.2595,   2.9883,  -5.2908,
          -3.6800,   0.3287,   5.0647,   7.0050,   3.3243,   2.3449,   1.0444,
         -15.7648,  -0.7784,   0.8343,  -1.5378,  -0.4195,   5.8323,  -0.5500,
           1.9066,  -0.6384,  -2.2009,   1.3492,  -1.8566,   1.3973,  -2.6040,
         -10.8060,  -0.8605,  -0.5270,  -0.6201,  -1.6238,  -0.1441,  -3.0862]],
       device='cuda:0', requires_grad=True)
```

Thus, I think the generated images look the same because the parameter theta which is transform alpha is too big.

Did you normalize the parameter theta in the training?

**Comment by LoreGoetschalckx** [comment]
Hi, I asked about the alpha value because I thought perhaps you picked a more extreme value than we did in the paper (0.1), but it is now clear that this is not the case. My intuition at this point is that the training wasn't necessarily unsuccessful. Rather, I would recommend generating test results for an earlier checkpoint. The EmoNet results from the paper were generated based on 40K training examples, not 400K. We noticed that training for "too long" often resulted in very extreme modifications.

About Question 3: If you want to try the memorability and aesthetics model, you can find them here: https://github.com/LoreGoetschalckx/GANalyze/tree/master/tensorflow/assessors (in TensorFlow). I am looking into whether we can release PyTorch versions as well in the future.

**Comment by UdonDa** [comment]
I used [-0.3, -0.2, -0.1, 0, 0.1, 0.2, 0.3] as the alpha.

>The EmoNet results from the paper were generated based on 40K training examples, not 400K.

I can generate the correct samples!!
Thanks!!
